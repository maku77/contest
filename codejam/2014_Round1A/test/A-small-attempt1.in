100
1 2
11
10
8 6
111001 010110 111101 101100 010010 110010 101000 110110
101101 110111 101001 100010 001001 001101 110011 100110
4 10
0000100100 0000111000 1110000111 1110011011
0000100100 1110011011 1110000111 0000111000
10 10
0100011101 0111111010 1101000100 1001100011 0000100010 0010101011 1110010100 1111010100 0010001001 1001111111
1100000010 0110111100 1110010010 1000111001 0011001111 0101011011 1111010010 1000100101 0001100100 0011101101
8 6
111000 001110 111111 101111 111101 011110 101101 101000
001001 101010 111000 101000 111111 101111 011001 111010
4 3
110 001 000 111
100 101 010 011
3 4
1101 1110 1010
1001 0111 1110
6 4
0100 1000 0101 1011 1010 1001
1001 0100 0101 1000 1011 1010
4 10
1100100010 0010111111 1000100010 0110111111
1000011110 0110000011 0010000011 1100011110
8 6
010111 110111 101111 101001 001001 001111 000001 100001
001010 000010 111100 000100 011100 101010 100010 100100
4 5
00110 11001 11010 11100
00000 00101 00011 11111
4 4
0101 1110 1101 0000
0101 1110 0110 1011
10 10
0000100110 0001010011 1101100111 1100111001 1000011100 0111001000 0010011010 0111010101 1010100101 1111101010
1110110110 0011110111 1100111011 0100000100 0001001011 0100011001 1001110100 1111101000 0010000010 1011001101
3 6
001001 001101 001010
001101 100100 101100
6 5
01111 00000 01001 00101 10010 11011
10011 11000 10100 00001 01101 01010
10 10
0000010100 1111011111 0000001101 0001001101 0111111000 1000100011 1110111010 1110110010 0101000000 1011100111
1101111110 1100010011 1000101011 1101110110 0010000001 0011011000 0110001100 0011000001 0100110100 1011101111
2 3
111 110
110 111
2 7
1110110 0100110
1111100 0101100
10 10
0111111111 0011100110 1010101011 1101010001 0001011010 1100101100 1000101010 1110000001 0000010101 0111010100
1101100001 0001101010 0000010111 1011101111 1111011101 1100101110 0110010000 0010111010 0100010001 1011000100
8 6
110000 111011 100010 111010 111000 101010 110011 110010
011011 000001 000010 001011 010011 001001 000011 001010
2 7
1001001 0010010
0001101 1100001
10 10
1110000001 1111110101 1110011011 0101101100 0010100110 1000110010 0001001000 0000000111 0101011000 1011111111
1010100010 1001101111 0011111011 0100110001 1110101000 1111011100 0001011010 1111000110 0000010101 0100000101
4 9
100111011 101110101 101110111 100001001
011000001 010001111 010001101 011110011
10 10
0000111001 0011011111 1101000000 0110111011 1011110110 1111001000 1100010111 0110100011 1001100100 0000001100
0100111100 0100001001 0010010011 1001110000 0111101111 1000100111 1011111000 1111000110 1101010100 0010001011
10 10
0111100000 0000010111 1101110111 0101010000 0000111101 1010001001 1010010110 1111101110 1010001000 0101101011
0011011110 1011000010 0001000111 1110111110 0100100000 1110010100 0100111111 1001001001 1011111001 0100100001
10 8
00110101 00111101 00010110 00101000 00011110 01011100 10010010 10011010 01010100 00100000
01100001 11000110 00001000 01110100 01000010 01001010 11001110 01111100 00000000 01101001
10 10
0111100000 1010010100 1111111111 0100000001 0100110000 1110001110 1001011001 0001111111 0000100011 1011001110
1010011110 0100100111 0011100100 0010111000 0111000110 1100111000 1000001001 1101001001 0111110111 1001010011
4 4
1010 1101 1110 0100
0110 0101 1100 0010
10 10
1110000110 1101111010 1010100101 0101001001 0101100110 0010110101 0001010001 1100001010 1011111010 0010011101
0100110000 1101011100 0011000000 1001101011 0010011111 0110111100 1010100111 1010001111 1101110011 0101000000
5 9
110001001 001100101 011011001 011001100 110011101
001110101 011011100 110001101 011001001 110011001
2 6
100010 000101
010010 110101
3 3
010 001 100
001 111 100
10 10
1111001001 0001110100 1101101001 1000010010 0111001001 1100001010 0010100101 0000011110 1111110110 0010110111
1101100010 1100000001 1111000010 1111111101 0001111111 0111000010 1000011001 0000010101 0010111100 0010101110
10 10
0101010110 0010100101 1000111011 0110100100 1111011000 1010001010 0111100111 0100100000 1001011001 1001011111
1010011010 1000011110 1011101100 0111100011 0001100010 1100011111 0111100101 1001011101 0110000001 0100110000
10 10
1000101100 0100011011 0101111111 0011011100 1010100001 1100110000 0111100110 1001011101 1111000011 0010000010
1010110110 1100101111 0100001100 0110000001 0001101110 0111110000 1101110001 1001001011 1011010010 0010011101
10 10
1001111100 0001000011 1010010111 0111111111 1001000000 1000101001 0100011000 1110001110 0110100000 0111110111
0011101110 1011010001 1100100011 1010000100 0100001101 0110110101 1011101101 0101010010 1000111010 0101011010
10 10
1000010111 0000000101 0111101000 0000111111 0011011000 1110001110 1011001001 1100100101 1111110010 0101110010
0000110011 1100010010 1001010101 0010101001 1000101001 0111100100 1111001100 1011111110 0100000011 0111011110
10 10
1000010000 0101100000 0111111101 1000011110 1110011010 0001000101 1010100011 1010100101 0101111011 0111001110
1010110011 1000001110 0111011110 0001010101 0101110000 1010110101 1000000000 0111101101 0101101011 1110001010
10 10
1001001100 1000110110 1111011111 0010010000 1110111011 0000001001 0101001011 1110100011 0101110100 0011100100
0011110001 0000011100 1000100011 1111001010 0010000101 0101100001 1110110110 1001011001 0101011110 1110101110
4 8
10010110 01101101 01011011 10011110
00011101 11100110 11101110 00101011
2 5
01100 10101
01110 10111
2 9
100111100 000111100
011100100 111100100
1 7
1000000
1100100
8 5
01000 01001 11010 11000 11100 11111 11011 10000
11110 11101 10101 11010 11111 11001 01101 01100
5 6
011000 110100 101101 110000 010110
011011 010101 111101 100000 111001
8 4
1000 0011 1100 0110 1001 0101 1110 0100
1011 1110 0011 0100 1001 1111 0010 0001
10 10
1001011000 0000010010 1110001101 1000101001 0001111010 1111010101 0011100011 0111000110 1100101111 0110110100
1010000010 1110100111 0010010001 0011001001 1100111110 1011110000 1101010110 0001101011 0101101101 0100011100
6 9
011001000 100100111 011111101 110101110 100101010 011011000
111001001 000010011 111000100 101000000 000100110 000110110
10 9
101011110 100010110 111101110 011101011 000100011 110100110 001111011 000110011 001101011 010100011
100000101 111001101 101001101 100010101 000110000 011001000 101011101 110000101 010000000 001111000
6 8
10100110 11101011 10001000 00000111 10111101 00111001
00010011 10011100 10100010 00111101 00100110 01110000
4 4
1110 1011 1010 0011
0110 1001 1110 1011
10 8
00111111 01010101 00000111 10111011 01110101 10101110 10010110 11101001 11111001 01001010
01110100 01010111 11010110 10010100 01110010 01101001 00101101 00111011 00100111 11010000
10 10
0110100101 0011001010 1001011010 1111110110 1000110100 0101001001 1000000001 0011101010 0100110111 1110011101
1010111011 0111110011 1011000101 0001101111 0000000100 1100111000 0111000110 1100011000 0110101000 1001010111
6 5
10000 01100 11011 11000 01111 10011
01011 10100 00011 10111 00000 01000
3 5
11100 11111 01110
10010 10011 11100
4 4
1110 1001 1100 1011
0011 0100 0110 0001
7 9
010110011 011101101 111111111 001110110 111101110 110110001 001111101
001100110 101101111 011111011 111000100 100100111 011100000 110111110
4 8
01100010 01001011 10111010 00010010
10010100 11001101 10111101 01100101
10 8
01000001 01110111 10011011 11100100 00000101 00010010 10010011 01111101 01001100 11010110
00001000 10010001 00000010 10100011 00111001 00110100 01110000 11100110 11101110 01100111
2 3
011 111
111 011
10 10
1001111101 0101001110 1110110011 1111011101 0000111010 1010000111 0111000100 0000100000 0101111011 1010000000
0101110110 1001110000 0110001011 1100110001 1011001111 1001000101 0110001100 0011010110 1100101011 0010111000
10 10
0010100000 0010010010 1010010101 0101101011 1111011101 1001110101 0101001010 1001101100 1110110110 0100001011
0111111000 0010110000 1000000110 1001100110 0011011011 1000100111 1111001101 0100000001 0100011000 1111111111
8 6
000010 011011 101010 000111 100001 010011 100010 100000
101010 111111 010011 000100 110101 111010 001111 011010
10 10
0010101000 1001011001 1110110011 1001110100 0110101110 0111011001 0011001111 0100000111 1000100000 1101010110
1111001100 1111100001 0000111011 1110110101 1011000011 0010010010 0001001100 0100111101 0101011010 1000100110
10 10
1000100011 0010100101 0101001101 1101001000 0111010110 1010111010 1110001110 0011111000 1001110101 0100010011
0011010000 0010000110 1101110011 1001011101 1110110110 0111101101 1000000000 1111101000 0100101011 0000011111
2 7
0001011 0100001
1110101 1011111
2 7
1011000 1100110
1000011 1111101
4 9
010011110 110101010 001001101 001011010
111000111 010100100 010001000 101011110
10 10
0010011000 0101101111 1000010001 0101110110 1010000111 0111100111 0111101010 1001111101 1010010000 1100001000
1011000100 0110110011 0100111011 1001000101 1101011100 1011010011 0100100010 0110111110 0011001100 1000101001
10 10
0000101110 0101011100 1001100110 0011101001 1110010011 1001000101 0000000010 0110010100 1111111001 1110111011
1010001101 0001101010 0110010100 1111111111 0001000010 1111010011 1001000101 0000101000 0110110111 1100111000
10 10
0011011100 1101011111 0000010011 0110111100 1000100001 1000100110 0111001010 1111111001 0010100011 1101000100
0111011111 1000111000 0111000100 1100100111 1101010001 0010111010 1001000111 0101100010 0010111101 1010001000
10 10
0010110111 0001001000 0010010001 0110001110 0101110111 1110111010 1001000011 1111011000 1101101100 1000100101
1011101100 0011011000 0101000111 1000010101 1111110011 1111010101 0100100001 1100101010 0010111010 0000001110
6 9
000110010 011111000 100010100 110000101 011110001 010000000
110101110 101100100 110100111 111010110 001000010 011010011
10 10
1000001100 0011001100 0011110110 1101100011 0110011011 0110110001 0001000000 1100110100 1001101111 1110011011
1110101111 0011011011 0111100011 1001110100 0001110100 1100011001 0110000000 0010001100 1001011110 1100100011
3 9
110100010 110010000 100111010
011100110 101001100 011010111
2 10
0011000010 1111101101
1111111101 0011010010
4 10
0101001110 0111111001 0111011110 0101101001
1010100001 1000110001 1010000110 1000010110
10 10
0000101011 1111100100 0100110111 1011010001 0011000010 1111110010 0001001010 0110001101 1000111101 1100011100
1011101001 1011111111 0010010110 1100100110 0111011001 0101010001 1111001010 1000000111 0000101100 0100110000
2 7
1100100 1101010
1110100 1111010
10 10
0000010010 1000100110 1110101111 1111001100 0000110100 1001011101 0111011001 0011100011 0100111010 1111000001
1011001100 1111100100 0110101011 1111000010 0001011001 0111010000 0000110111 1100010101 0000111010 1000101111
10 10
0101110010 1011000001 0011000111 1100110110 0000111010 1100011101 0011101000 1110001010 0001101101 1110010101
1001100110 0110111000 1011011010 0110010111 1110010001 1001001101 0100111101 1011000101 0000100010 0101101010
3 7
1101011 0010100 1001011
0111011 1000100 0011011
1 5
00110
10100
4 5
00001 11110 00101 11010
10101 10001 01010 01110
3 7
0111110 0010001 1111111
1011000 0111010 0100011
8 9
001010000 000001000 100100001 001000010 110001010 110010000 100100101 000111010
010001111 101100011 000111111 001010010 110110111 000100000 000100001 110000111
5 8
10011001 01110010 01100101 11101011 01011101
01110101 11111011 10001001 01001101 01100010
6 10
0000111010 1000101010 1111010100 1000111010 0111010100 1111000100
0000111011 0111010101 0000101011 0111000101 1000101011 1111000101
3 6
010110 101110 101010
110100 011110 000101
10 10
0110110001 1101101011 0101000110 0110010111 0000001111 0001001100 1110010001 0010001110 1111111010 1001000000
0000011010 1001001111 0011000000 1110101111 1011010000 0010110100 1010001100 1101001011 1010100111 1110101001
10 10
0011010111 0001011001 1010000101 0100110011 1100000110 1000001000 1010111000 0111101011 1101101110 0111110100
0010101101 0000011101 0111001011 1101010001 0000100000 1101001110 1110010110 1011111100 1001110010 0110100011
10 10
0101111000 1111111011 1100000111 0010000101 1001000110 1101000100 0011010000 0010101011 0110111010 1000111101
1011101101 1100111110 0111010011 1100010000 0110101000 1000101111 1101000101 0010010010 0001101110 0011010001
10 10
0010000000 0000011111 0111000010 1101111100 1110100011 0100000101 1011011001 0000110001 1111111110 1001101110
0011011100 0010000001 0110100110 1101001110 0000000011 1101100000 1001111010 1010111101 1111111111 0100010001
2 9
010001011 100100001
100110001 010011011
8 9
101010001 011010001 100010101 001101000 111000001 000101100 010010101 110000101
001101011 111000010 000101111 100010110 011010010 110000110 101010010 010010110
2 10
0010001100 1101100001
0100111010 1011010111
8 6
010110 011101 110011 001011 111110 111010 110001 101011
110011 101001 111100 010100 110001 001001 011111 111000
6 10
1000010110 0110011110 1110111111 1100111110 0111000111 1010011000
0101000000 1100111000 0111000001 1101100001 0001100111 0011101001
6 8
00011101 10011000 00000111 10000010 11101000 01101101
10010010 11111000 01100111 00010111 01111101 11100010
4 6
100000 100010 101010 101000
101000 100010 101010 100000
