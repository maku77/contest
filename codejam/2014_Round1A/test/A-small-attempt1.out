Case #1: 1
Case #2: 4
Case #3: 0
Case #4: 3
Case #5: 3
Case #6: 1
Case #7: NOT POSSIBLE
Case #8: 0
Case #9: 4
Case #10: 3
Case #11: 3
Case #12: 3
Case #13: 4
Case #14: NOT POSSIBLE
Case #15: NOT POSSIBLE
Case #16: 4
Case #17: 0
Case #18: 2
Case #19: 7
Case #20: 3
Case #21: NOT POSSIBLE
Case #22: 5
Case #23: 7
Case #24: 3
Case #25: 6
Case #26: 3
Case #27: 5
Case #28: 1
Case #29: 5
Case #30: 1
Case #31: 2
Case #32: 2
Case #33: 3
Case #34: 7
Case #35: 7
Case #36: 5
Case #37: 7
Case #38: 1
Case #39: 3
Case #40: 3
Case #41: 1
Case #42: 4
Case #43: 2
Case #44: 2
Case #45: 3
Case #46: 3
Case #47: 4
Case #48: 6
Case #49: 4
Case #50: 5
Case #51: NOT POSSIBLE
Case #52: NOT POSSIBLE
Case #53: 7
Case #54: 2
Case #55: NOT POSSIBLE
Case #56: 1
Case #57: NOT POSSIBLE
Case #58: 7
Case #59: 5
Case #60: 0
Case #61: 5
Case #62: 7
Case #63: NOT POSSIBLE
Case #64: 5
Case #65: 5
Case #66: 3
Case #67: 3
Case #68: NOT POSSIBLE
Case #69: 3
Case #70: 6
Case #71: 6
Case #72: 5
Case #73: 5
Case #74: 9
Case #75: NOT POSSIBLE
Case #76: 1
Case #77: 5
Case #78: 5
Case #79: 1
Case #80: 8
Case #81: 3
Case #82: 2
Case #83: 2
Case #84: 1
Case #85: NOT POSSIBLE
Case #86: NOT POSSIBLE
Case #87: 1
Case #88: 3
Case #89: NOT POSSIBLE
Case #90: NOT POSSIBLE
Case #91: 5
Case #92: 6
Case #93: 9
Case #94: 1
Case #95: 2
Case #96: 6
Case #97: 1
Case #98: 9
Case #99: 5
Case #100: 0
